in-toto-spec.md

```diff
@@ -760,6 +769,12 @@ match any materials.
   other words, this path must appear as a material and a product within this
 step and their hashes must not match.
 
+The artifact rules contained in the `"expected_materials"` and
+`"expected_products"` fields operate in a similar fashion as a firewall. This
+means that the first rule that matches a specific artifact in the link metadata
+will be used to match that artifact. Likewise, there is an implicit `ALLOW *`
```

Comment:
We opted for an implicit ALLOW based on the feedback on people (it's mostly a bet for usability), although I understand the security concern.

Reply:
I don't feel strongly that we should use one over the other between ALLOW * or DENY *.
It's ok to choose one over the other if you have a good reason.

However, if you choose ALLOW *, we should not say "Likewise, ...", because this implies that firewalls also work like that. Instead, I suggest to say "In addition, there is an implicit ALLOW * at the end of such fields."

Reply:
I changed the wording for this